jdbc/src/java/org/apache/hive/jdbc/HiveConnection.java

```diff
@@ -19,6 +19,7 @@
 package org.apache.hive.jdbc;
 
 import com.google.common.annotations.VisibleForTesting;
+import org.apache.hadoop.hive.metastore.conf.MetastoreConf;
```

Comment:
This import seems un-necessary. Can be removed. 
I did not like the idea of adding a dependency on this class into the Hive JDBC module but looks like its not needed.

Reply:
Weird, I did not remember do that. Maybe the editor help me? I will remove it.